dig/default.go

```diff
@@ -20,30 +20,30 @@
 
 package dig
 
-// Default graph is used for all the top-level inject calls
-var defaultGraph = newGraph()
+// DefaultGraph is used for all the top-level inject calls
+var DefaultGraph Graph = newGraph()
```

Comment:
I envisioned testing other packages: constructors can accept an option to use a different graph to resolve objects, but what is the default graph for them?
For example, for uhttp.Client, we can load auth/tracer from a default graph, so users will not need to worry about parameters. But how would we test for auth errors? By injecting the injector(with options)..
An example, where a default global is used is http.DefaultClient.